interop-cats/shared/src/main/scala/zio/interop/cats.scala

```diff
@@ -184,9 +184,7 @@ private class CatsConcurrentEffect[R](rts: Runtime[R])
             _ => fa
           )
           .interruptible
-          .nonDaemon
-          .forkInternal
-          .daemon
+          .forkDaemon
```

Comment:
I think `.forkWithErrorHandler(_ => ZIO.unit)` is the replacement for `forkInternal`. @adamgfraser can you confirm?

Reply:
With this replacement, tests hang forever

Reply:
what about `.forkWithErrorHandler(_ => ZIO.unit).tap(_.disconnect)`?

Reply:
`.forkWithErrorHandler(_ => ZIO.unit).tap(ZIO.disown)` seems okay